api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -148,6 +171,33 @@ message DownstreamEnforcing {
   }
 }
 
+// Enforce faults on upstream redis cluster.
+message RedisUpstreamEnforcing {
```

Comment:
@snowp Did you have any opinions on whether redis should be a new enforcer type or not?

Reply:
Intuitively I'd say not, enforcer is more about which set of envoy instances should be enforcing the fault which should be independent on fault type